### PR TITLE
Feature/pallet

### PR DESCRIPTION
**Pull request by Feetjexx**
pallet model and texture added to prefab

### CODE REVIEW

**Review by Joost-Robat — APPROVED** [review]
Looks good. Ready for merge IMO.

File: Assets/Art/2D/Textures/Pallet/T_Pallet.png
Comment (TryFox) [inline Assets/Art/2D/Textures/Pallet/T_Pallet.png:1]: Add _D suffix

File: Assets/Art/2D/Textures/Pallet/T_Pallet.png.meta
Comment (TryFox) [inline Assets/Art/2D/Textures/Pallet/T_Pallet.png.meta:1]: Add _D suffix

**Review by TryFox — CHANGES_REQUESTED** [review]
Add _D suffix

**Review by TryFox — APPROVED** [review]

**Pull request merged**